lib/moog-require.js

```diff
@@ -21,27 +20,28 @@ module.exports = function(options) {
   self.improvements = {};
 
   if (self.options.bundles) {
-    _.each(self.options.bundles, function(bundleName) {
+    for (const bundleName of Object.values(self.options.bundles)) {
```

Comment:
Isn't it just an array, why Object.values here?

Reply:
indeed